### PR TITLE
Javascript primitives and variables

### PR DESCRIPTION
**Pull request by sararoohan**
image

### CODE REVIEW

File: javascript-primitives-and-variables/main.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+var totalPets = 0;
+var unknown = null;
+var never;
+
+console.log('value of Full Name:', fullName);
+console.log('value of Is Cool:', isCool);
+console.log('value of Total Pets:', totalPets);
+console.log('typeof fullName:', typeof 'Sara Roohan');
+console.log('typeof isCool:', typeof true);
+console.log('typeof totalPets:', typeof 0);
```
Comment (BrettAlbright) [inline javascript-primitives-and-variables/main.js]: you want too place the actual variables here in the console logs.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
small update!

File: javascript-primitives-and-variables/main.js (hunk 1 above)
Comment (sararoohan) [reply javascript-primitives-and-variables/main.js]: Updated!
image

**Review by BrettAlbright — APPROVED** [review]
👍

**Pull request merged**